### PR TITLE
[NUI] Add Visuals(Public) TCs.

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
Add public/Visuals/* TCs.

### API Changes ###

 - ACR:

### CODE REVIEW

File: test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Visuals/TSAnimatedImageVisual.cs [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+{
+    using tlog = Tizen.Log;
+
+    [TestFixture]
+    [Description("public/Visuals/AnimatedImageVisual")]
+    public class PublicAnimatedImageVisualTest
+    {
+        private const string tag = "NUITEST";
+        private string image_path = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "picture.png";
+        private static bool _flagComposingPropertyMap;
```
Comment (huiyueun) [inline test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Visuals/TSAnimatedImageVisual.cs:18]: delete '_'.

Comment (AchoWang) [reply test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Visuals/TSAnimatedImageVisual.cs]: Done.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by huayongxu — APPROVED** [review]
lgtm.

**Pull request merged**